lib/ansible/modules/network/files/net_put.py

```diff
@@ -49,7 +49,7 @@
         present in the src file. If mode is set to I(binary) then file will be
         copied as it is to destination device.
     default: binary
-    choices: ['binary', 'template']
+    choices: ['binary', 'text']
```

Comment:
This change should be reflected in the description as well, which still references `template`.